docs/hub/enterprise-hub-gating-group-collections.md

```diff
@@ -0,0 +1,166 @@
+# Gating Group Collections
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise">Enterprise Hub</a>.
+</Tip>
+
+Gating Group Collections allow organizations to grant (or reject) access to all the models and datasets in a collection at once rather than repo per repo. Users will only have to go through **a single access request**.
```

Comment:
```suggestion
Gating Group Collections allow organizations to grant (or reject) access to all the models and datasets in a collection at once, rather than per repo. Users will only have to go through **a single access request**.
```

maybe?